Automattic/wp-calypso
# Comments: remove feature flags

Cleaning up feature flags that are enabled in all environments.

### Testing instructions

1. Smoke test the `comments/all` section and verify that everything is working as expected.
2. Make sure that there are no remaining usages of these flags in the code.
